web_src/js/components/RepoActionView.vue

```diff
@@ -403,18 +403,9 @@ const sfc = {
 
     // show at most one of log seconds and timestamp (can be both invisible)
     toggleTimeDisplay(type) {
-      const toToggleTypes = [];
-      const other = type === 'seconds' ? 'stamp' : 'seconds';
       this.timeVisible[`log-time-${type}`] = !this.timeVisible[`log-time-${type}`];
-      toToggleTypes.push(type);
-      if (this.timeVisible[`log-time-${type}`] && this.timeVisible[`log-time-${other}`]) {
-        this.timeVisible[`log-time-${other}`] = false;
-        toToggleTypes.push(other);
-      }
-      for (const toToggle of toToggleTypes) {
-        for (const el of this.$refs.steps.querySelectorAll(`.log-time-${toToggle}`)) {
-          toggleElem(el, this.timeVisible[`log-time-${toToggle}`]);
-        }
+      for (const el of this.$refs.steps.querySelectorAll(`.log-time-${type}`)) {
+        toggleElem(el, this.timeVisible[`log-time-${type}`]);
```

Comment:
See the comment above: `    // show at most one of log seconds and timestamp (can be both invisible)`

Reply:
Comment removed.